<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊城市鲁昌保安服务有限公司信息管理系统</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>801</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>112</width>
           <height>357</height>
          </rect>
         </property>
         <attribute name="label">
          <string>公司相关</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>77</width>
            <height>201</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonCompany">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>公司简介</string>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInstitution">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>机构设置</string>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRules">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>规章制度</string>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonLaw">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>法律法规</string>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPhoneBook">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>通讯录</string>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>28</height>
          </rect>
         </property>
         <attribute name="label">
          <string>保安服务</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>0</y>
            <width>77</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonGuardInfo">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>人员信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonGuardCompanyInfo">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>客户单位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonGuardPosition">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>岗点分布</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>28</height>
          </rect>
         </property>
         <attribute name="label">
          <string>物业服务</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>77</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButtonPropertyInfo">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>员工信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPropertyCompany">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>客户单位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPropertyUse">
             <property name="font">
              <font>
               <family>黑体</family>
              </font>
             </property>
             <property name="text">
              <string>物资领用</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>28</height>
          </rect>
         </property>
         <attribute name="label">
          <string>财务管理</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>28</height>
          </rect>
         </property>
         <attribute name="label">
          <string>技防服务</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="page_6"/>
        <widget class="QWidget" name="page_7"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
